### ISSUE TITLE
Is it possible to "remove" Users from the User table

### ISSUE THREAD

**Issue by laradg** [laradg/copra-todo#66]
There are a few approvers who no longer work with COPRA.
Is it possible to set the "state" column of their User entry in the SQL DB to something so that they don't show up in the "Approvers list" in AVMT?

**Comment by davedx** [comment]
Yep. I've made a change so if their user state is INACTIVE, they will no longer be able to login or reset their password.

**Comment by laradg** [comment]
Thank you for this. The INACTIVE user is still showing up in the Approvers list in AVMT
The list I am referring to can be found by:
- Going to AVMT
- Select a Professional in the list
- Edit one of the projects in "Project details"
- In the list of possible approvers (left), the INACTIVE approver should no longer be visible

Thanks!

**Comment by davedx** [comment]
Fixed.

**Comment by laradg** [comment]
The solution for this issue has caused a bug, when the "deleted" User is still set as Approver for a project, that project can no longer be opened in CMT.

Please resolve by removing all entries for this userID in AVProjectApprover.
Thanks!
